gmodule.lua

```diff
@@ -118,23 +117,21 @@ function gModule:__init(inputs,outputs)
    -- computation on the graph is done through topsort of forward and backward graphs
    self.forwardnodes = self.fg:topsort()
    self.backwardnodes = self.bg:topsort()
-   self.modules = {}
```

Comment:
I believe you did not mean to delete this while clearing merge conflicts.
Line 121 to 127 were added in https://github.com/torch/nngraph/pull/56/files